components/sample-data-service/README.md

```diff
@@ -0,0 +1 @@
+# Sample data service
```

Comment:
Could we add a bit of content here? What does the service do? How do you use it?